bin/new.bat

```diff
@@ -0,0 +1,76 @@
+::  new.bat
+::  Creation API
+::
+::  This file is called whenever a user creates a copy of templates, such as: new project p999_Meindbender_Sync
+::
+::  Arguments:
+::    %1: template to copy
+::    %2: name to rename the template
+::
+::
+
+@echo off
+
+if "%1"=="" goto :usage
+
+if "%2"=="" goto :missingname
+
+if "%1"=="project" goto :copyProject
+
+if "%1"=="asset" goto :copyAsset
+
+if "%1"=="shot" goto :copyAsset
+
+if "%1"=="maya" goto :createMaya
+
+if "%1"=="nuke" goto :createNuke
+
+:usage
+echo Enter what you want to create
```

Comment:
Elaborate on this 

Provide detailed instructions along with examples of how to use it. Picture someone in Brazil encountering this for the first time and not having been told how to use it. He should be able to understand the instructions on his own